Source/Reporting/Web/src/app/Reporting/CaseReportsForListing/training-case-reports/training-case-reports.component.html

```diff
@@ -10,20 +10,20 @@ <h3>
 <div>
         <a routerLink="../../list/all" > {{ 'REGISTER.REPORT.report_name' | translate }}</a>
     </div>
-
-<div class="paging row">
-    <div class="col-md-7 paging-filters">
-        {{ 'REGISTER.REPORT.filter' | translate }}:
+<div class="row">
+    <div class="col-md-4 paging-filters">
         <span *ngFor="let filter of allFilters">
             <button
                 class="small {{ filter.cssClass }}"
                 [ngClass]="{'current': this.currentFilter === filter}"
                 (click)="clickFilter(filter)">
-                {{filter.description}}
+                {{filter.description}} <!-- Translate-->
```

Comment:
I think this should be done now. And not added as a comment. I can point you to the correct translations if they aren't already in the files.

Reply:
I'll see what I can do about that